docs/glossary.rst

```diff
@@ -0,0 +1,102 @@
+
+    Default Actions
+        Built-in actions that carry a special meaning to Rasa.
+
+    Domain
+        A domain lists all the intents, entities, and actions of an assistant.
+
+    Entities
+        Structured information that can be extracted from a user message (e.g. telephone number).
+
+    Event
+        Conversations in Rasa are represented as a sequence of events. 
+
+    Forms
+        One of the most common conversation patterns is to collect a few pieces of information from a user in order to do something (book a restaurant, call an API, search a database, etc.). This is also called slot filling.
+
+    Intent
+        Meaning of a users message selected as one of a set of predefined meanings (e.g. greeting).
+
+    Interactive Learning
+        You provide feedback to your bot while you talk to it. This is a powerful way to explore what your bot can do, and the easiest way to fix any mistakes it make.
+
+    NLG
+        Natural Language Generation (NLG) is the component that generates natural language output from structured data.
+
+    NLU
+        Natural Language Understanding (NLU) deals with parsing and understanding human language into a structured format. Rasa NLU is the part of Rasa that performs intent classification and entity extraction.
+
+    Open Source Software
+        Software with source code that anyone can inspect, modify, and enhance.
+
+    Pipeline
+        A Rasa bot's NLU system is defined by a pipeline, which is a list of NLU components in a particular order. A user input is processed by each component one by one before finally giving out the structured output.
+
+    Policy
+        Policies make decisions on how conversation flow should proceed. A Core model can have multiple policies included, and the policy whose prediction has the highest confidence decides the next action to be taken.
+
+    Rasa Core
+        The dialogue engine that decides on what to do next in a conversation.
+
+    Rasa Core policy
+        Decides which action to take at every step in the conversation. There are different policies to choose from, and you can include multiple policies in a single agent. At every turn, the policy which predicts the next action with the highest confidence will be used. If two policies predict with equal confidence, the policy with the higher priority will be used.
+
+    Rasa NLU component
+        Allows you to customize your model and finetune it on your dataset. Incoming messages are processed by a sequence of components called a pipeline. These components are executed one after another in a so-called processing pipeline. There are components for entity extraction, for intent classification, pre-processing, and others.
+
+    Repository
+        A folder for your project. Your project's repository contains all of your project's files and stores each file's revision history. You can also discuss and manage your project's work within the repository.
+
+    Skill
```

Comment:
we don't use 'skill', this is a user goal, see https://rasa.com/docs/rasa/dialogue-elements/dialogue-elements/